docs/slack.rst

```diff
@@ -94,21 +70,65 @@ We have some helpful Slackbot responses that you can summon in any channel, at a
 
   - Returns the answer for one of our most frequently asked questions: why you can't access the full chat history in the Write the Docs Slack.
 
+Channel guide
+-------------
+
+There are lots of channels. Here's an overview of interesting channels and how to find more.
```

Comment:
Nit pick, sorry.

How about splitting out "these are the default channels everyone is added to", and then "here are some interesting / topical channels"

Reply:
Sorry @ddbeck I'll be more clear. I'm super happy with your structure, I just find the mention of "interesting ones" just before the list of defaults, even in the overview, a bit jarring.

Your new structure of separating out the default ones I think is even better though.

And let me apologize again for the nitpick

Reply:
@plaindocs I tried my new proposed outline and was sort of disappointed in it (it also seems jarring to have default channels _not_ in the channel guide). I put it back, then rewrote the intro. What do you think?

Reply:
Love it. :shipit: